packages/angular_devkit/architect/node/node-modules-architect-host.ts

```diff
@@ -129,7 +129,7 @@ export class WorkspaceNodeModulesArchitectHost implements ArchitectHost<NodeModu
 
     const projectDefinition = this._workspace.projects.get(projectName);
     if (!projectDefinition) {
-      throw new Error('Project does not exist.');
+      throw new Error('Project does not exist: ' + projectName);
```

Comment:
Can we change this to `Project "${projectName}" does not exist.` please?

Reply:
done :-)